test/boards/led_buttons-1/led_buttons-1.pro

```diff
@@ -0,0 +1,62 @@
+update=22/05/2015 07:44:53
+version=1
+last_client=kicad
```

Comment:
Someone secretly using kicad? ;-) Nice.

Reply:
Hah, using both. This file shouldn't be here. sorry